content/blog/demodscclass-your-first-class-based-dsc-v3-resource.md

```
@@ -0,0 +1,336 @@
+> To see tracelogs add the argument `--trace-level trace`, e.g. `dsc --trace-level trace resource get`. The different trace levels are: `error`, `warning` (default), `info`, `debug`, `trace`.
+
+### Known issues
+
+#### DSC executable
+
+Currently the get, test and set operation fails if there are any `Write-Verbose`, `Write-Warning`, or `Write-Debug` that return output. You will then get the error `ERROR JSON: expected value at line 1 column 1`.
+See issue [Output during module import or output from Get operaton interrupts the parsing process for class-based resource](https://github.com/PowerShell/DSC/issues/833).
+
+For example if we would add parameter `-Verbose` the above line `Write-Verbose -Message 'Get called'` then the get operation would fail with:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)